### PR TITLE
Rn2017.19

### PR DESCRIPTION
**Pull request by mystiberry-okta**
## Description:

First draft of RN for 2017.19

## Type of Pull Request:

- [x] Content (Documentation updates or typo-fixes)
- [ ] Blog Post
- [ ] Functional (CSS changes, JS updates, or layout modifications)

### Resolves:
* [OKTA-125465](https://oktainc.atlassian.net/browse/OKTA-125465)

## How Has This Been Tested?
- [x] I have built this locally and verified that it does not break existing functionality.
- [ ] For functional changes, I have tested against Chrome, Firefox, and Safari, and mobile.
- [ ] For functional changes, I have added tests.

## Primary Reviewers:

- *Tag reviewer(s)*
@rchild-okta

### CODE REVIEW

**Comment by mystiberry-okta** [comment]
@rchild-okta Is this going to happen every time I create a Doc PR in 'weekly'? It was created directly in weekly. What did I do wrong?

**Comment by rchild-okta** [comment]
It's going to happen whenever you base your branch off of master, and there is a difference between what is in master and what is in weekly.

For new commits that are not in the process of being migrated over from before we had weekly:

```bash
# Get weekly up to date
$ git fetch origin
$ git checkout weekly
$ git merge origin/weekly

# Create a topic branch that is based off of weekly
$ git checkout -b mb-okta-1234-some-descriptive-branch-name
```

For commits that are migrating over, you need to set the base to weekly:
```bash
$ git fetch origin
$ git rebase -i origin/weekly

# In the interactive editor, you will have the commits that are diverged
# because you are going back in time - master is ahead. So, you will
# need to remove those lines with commits that are not part of what
# you want to pull in.

$ git push -f origin {branchName}:{branchName} 
```

tldr; No, it is not going to happen every time. For future commits, if you branch off of weekly there is no difference from the old way of doing things except for using weekly as your starting point instead of master. For branches that are migrating over, just follow the above steps. The main point is to remove commits that have been added in master that are not yours.

**Comment by mystiberry-okta** [comment]
I have no idea how to branch off weekly. How do you do that? I did a git pull for master and then for weekly and then created a branch and didn't get this stuff. Is that what you mean, @rchild-okta thanks!

**Comment by mystiberry-okta** [comment]
So I am still in master when I start this, yes?
# Get weekly up to date
$ git fetch origin
$ git checkout weekly
$ git merge origin/weekly
thanks @rchild-okta

**Comment by mystiberry-okta** [comment]
@rchild-okta all fixed (thank you). Just waiting for Wed. morning to ensure no more changes are requested.
